pkg/smartcontract/parameter.go

```diff
@@ -197,7 +199,7 @@ func (p *Parameter) UnmarshalJSON(data []byte) (err error) {
 		if err = json.Unmarshal(r.Value, &h); err != nil {
 			return
 		}
-		p.Value = h
+		p.Value = h.Reverse() // Hash160 should be marshaled in BE but default marshaler uses LE.
```

Comment:
Not sure about this also.

Reply:
For `UInt160` here (https://github.com/neo-project/neo/blob/master-2.x/neo/SmartContract/ContractParameter.cs#L156) they use override `ToString()` method (https://github.com/neo-project/neo/blob/master-2.x/neo/UIntBase.cs#L126) which returns BE string

Reply:
But our internal storage is BE and theirs is LE IIRC, so reversing here gives you LE.